nodejs/week2/homework/meal-sharing/src/backend/api/reviews-router.js

```diff
@@ -0,0 +1,35 @@
+const express = require("express");
+const router = express.Router();
+
+const reviews = require("./../data/reviews.json");
+
+router.get("/", async (request, response) => {
+    try {
+        response.send(reviews);
+    } catch (error) {
+        throw error;
```

Comment:
throwing an error here is not helpful because it would crash your server app if it's not caught.
you can return an error code like `500 Internal Server Error` instead

Reply:
I just removed the try catch instead

Reply:
Yes understood done